tests/schema/test_execution.py

```diff
@@ -174,11 +175,13 @@ def example(self) -> int:
 
     schema = strawberry.Schema(query=Query)
 
-    query = """
+    query = dedent(
+        """\
         query {
             example
-        }
+        }\
```

Comment:
you could use `.strip` to skip the `\` 😊

Reply:
Nice catch! More pythonic

Reply:
Actually, this makes `dedent` to not work as expected because it strips all the leading `\n` and we only want 1 to be removed so dedent takes out evenly those break lines:

```text
>>> print(dedent("""
...     test
...
...     GraphQL request:2:5
...     1 | query {
...     2 |     example
...       |     ^
...     3 | }
... """.strip()
... ))
test

    GraphQL request:2:5
    1 | query {
    2 |     example
      |     ^
    3 | }
```

```text
>>> print(dedent("""\
...     test
...
...     GraphQL request:2:5
...     1 | query {
...     2 |     example
...       |     ^
...     3 | }\
... """
... ))
test

GraphQL request:2:5
1 | query {
2 |     example
  |     ^
3 | }
```

What am I missing?

Reply:
@scratchmex I think it should work if you put the `.strip()` on the result of `dedent`:

```python
>>> print(dedent("""
...     test
...
...     GraphQL request:2:5
...     1 | query {
...     2 |     example
...       |     ^
...     3 | }
... """
... ).strip())
```